working_groups/learning_journey.md

```diff
@@ -0,0 +1,81 @@
+## Goals
+The primary goal of the Learning Journey Working Group is to provide Nix users with adequate resources to learn Nix on their own. This means that it is the job of the WG to:
+- Ensure that adequate documentation exists
+- Ensure that the documentation is structured in an intuitive way
+- Ensure that the documentation is discoverable
+- Ensure that the documentation meets a minimum standard of quality
+- Establish guidelines for contributions
+
+By extension the WG will be responsible for practical aspects such as:
+- Some portion of the structure of `nix.dev` so that documentation is discoverable and organized intuitively
+- Ensuring that bare minimum authoring tools exist to keep the barrier low for contributions from new contributors
+
+## Membership
+Leads: @zmitchell
+
+Members:
+- @infinisil
+- TBD
```

Comment:
@olaf @asymmetric @wiseh @tonyfinn please add yourself as a suggestion if you're interested in participating.